website/content/gateway/api_reference/resources/authorizations/authorizations_methods/token.md

```diff
@@ -10,3 +10,97 @@ Two token frameworks are supported:
 
 * `token[m4m]`: Mastercard Digital Enablement Service (MDES) for Merchants (M4M)
 * `token[vts]`: Visa Token Service (VTS)
+
+Requirement of some parameters depends on the initiator of the transaction; a parameter might be required for cardholder-initiated transactions (CITs) and otherwise optional. See the details for each parameter.
+
+{{% notice %}}
+**Notice**: Signing is required to use the `token` payment methods.
+{{% /notice %}}
+
+#### Method: token[m4m]
+
+The required values are found in either the MDES `transact` response or the Secure Card on File (SCOF) `checkout` response.
+
+{{% description_list %}}
+{{% description_term %}}token[m4m][tan] {{% regex %}}[0-9]{12,19}{{% /regex %}}{{% /description_term %}}
+{{% description_details %}}Token Account Number (TAN) to charge.
+
+Found in `encryptedPayload.encryptedData.accountNumber` (MDES) or `encryptedPayload.token.paymentToken` (SCOF).
+{{% /description_details %}}
+
+{{% description_term %}}token[m4m][expire_month] {{% regex %}}[0-9]{2}{{% /regex %}}{{% /description_term %}}
+{{% description_details %}}Expiry month of token to charge.
```

Comment:
```suggestion
{{% description_details %}}Expiry month of TAN to charge.
```

?

(Same below for `expire_year`.)

Reply:
Nah, I think that's not right.

Reply:
Hmm, but should the description for `token[...][tan]` then read

> Token Account Number (TAN) of the token to charge.

?

Reply:
I think the latter suggestion matches what we seen in the API documentation for MDES, SCOF and VTS.

For MDES:

![Screenshot from 2022-09-29 20-41-39](https://user-images.githubusercontent.com/14069221/193116585-a2734fdc-ed37-4789-9d45-fd76638eb9af.png)

For SCOF: The names `tokenExpirationMonth` and `tokenExpirationYear` are used. So, again, it's about the token, not the TAN.

For VTS, from the publication "Visa Token Service - e-Commerce/Card-on-File Implementation Guide for Token Requestors":

![Screenshot from 2022-09-29 20-57-21](https://user-images.githubusercontent.com/14069221/193118951-6e61f007-278b-48ca-8739-db290be21bb1.png)

Reply:
https://github.com/clearhaus/gateway-api-docs/pull/149/commits/7449c7a058e591f5ff4ad17da92a583064453b72